clients/ctl/admin-ui/src/features/dataset/DataCategoryInput.tsx

```diff
@@ -57,7 +57,7 @@ const DataCategoryDropdown = ({
         Select data categories
       </MenuButton>
       <MenuList>
-        <Box maxHeight="500px" minWidth="300px" overflowY="scroll">
+        <Box maxHeight="50vh" minWidth="300px" overflowY="scroll">
```

Comment:
The cypress test was failing because the dropdown was larger than the remaining screen space within the cypress window (and so it couldn't click on the checkbox since it was technically off the screen). I'm not sure what the best way to fix this is, but giving it a proportion of the vh seems to work ok